<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetUSB202</class>
 <widget class="QWidget" name="WidgetUSB202">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WidgetUSB202</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_BP1">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(239, 41, 41);
</string>
           </property>
           <property name="text">
            <string>BP1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_LED1">
           <property name="text">
            <string>LED1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_BP2">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(239, 41, 41);
</string>
           </property>
           <property name="text">
            <string>BP2</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_LED2">
           <property name="text">
            <string>LED2</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_BP3">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(239, 41, 41);
</string>
           </property>
           <property name="text">
            <string>BP3</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_LED3">
           <property name="text">
            <string>LED3</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_BP4">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(239, 41, 41);
</string>
           </property>
           <property name="text">
            <string>BP4</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_LED4">
           <property name="text">
            <string>LED4</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_Quitter">
         <property name="text">
          <string>QUITTER</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Quitter</sender>
   <signal>clicked()</signal>
   <receiver>WidgetUSB202</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
